extensions/ql-vscode/src/view/variant-analysis/RawResultsTable.tsx

```diff
@@ -9,7 +9,7 @@ import { RawResultRow } from "./RawResultRow";
 const numOfResultsInContractedMode = 5;
 
 type TableContainerProps = {
-  columnCount: number;
+  $columnCount: number;
```

Comment:
See https://styled-components.com/docs/basics#attaching-additional-props for docs on props for styled components.